components/FormManual/FormManual.js

```diff
@@ -105,6 +164,51 @@ export default function FormManual({
             </button>
           </>
         )}
+
+        <StyledImageInputWrapper>
+          {(imageUploaded ||
+            (actionMode === "edit" &&
+              currentFlashcard?.imageUrl &&
+              showOriginalImage)) && (
+            <StyledImageWrapper onClick={handleImageClose}>
+              <StyledIconWrapper>
+                <MarkAsIncorrect />
+              </StyledIconWrapper>
+              <StyledImagePreview
+                src={
+                  imageUploaded
+                    ? URL.createObjectURL(image)
+                    : currentFlashcard.imageUrl
+                }
+                alt="Preview of the image"
+                sizes="300px"
+                fill
+              />
+            </StyledImageWrapper>
+          )}
+          <>
+            <StyledImageInput
+              $hidden={
+                imageUploaded ||
+                (actionMode === "edit" &&
+                  currentFlashcard?.imageUrl &&
+                  showOriginalImage)
+              }
```

Comment:
This is the exact same condition as the one above. Just use the `: <Upload Button>` strategy from the comment above instead of the $hidden attribute. By this, the button is only rendered if the statement is false.